.gitignore

```diff
@@ -1,6 +1,10 @@
 /docs/book/*
 /target
+/log
 Cargo.lock
+omnipaxos_core/rocksDB/
+omnipaxos_core/commitlog/
+clean_test.sh
```

Comment:
Would it be possible to add the functionalities of this script into the tests itself? Doesn't make sense to have a bunch of garbage after each test.

Reply:
Should be possible, the script only removes the `commitlog` and `rocksDB` directories before running `cargo test.` This can be done at the last test

Reply:
No, it should be done at every test that uses PersistentStorage because we might not always run all tests.